tests/listener_test.cc

```diff
@@ -192,3 +194,20 @@ TEST(listener_test, listener_bind_ephemeral_port) {
     Pistache::Port bound_port = listener.getPort();
     ASSERT_TRUE(bound_port > (uint16_t)0);
 }
+
+
+TEST(listener_test, listener_bind_ephemeral_v6_port) {
+    Pistache::Port port(0);
+    Pistache::Address address(Pistache::Ipv6::any(), port);
+
+    Pistache::Tcp::Listener listener;
+    if (not listener.systemSupportsIpv6()) {
```

Comment:
TIL: https://en.cppreference.com/w/cpp/language/operator_alternative
I had no idea that this was valid c++.

Reply:
No need to remove it, its valid.  Just a funny comment...

Reply:
It's something I've carried across from my heavy use of Python in other projects, I find it makes it easier to read when skimming over a large block.